web/storagenode/src/app/components/VChart.vue

```diff
@@ -53,21 +53,53 @@ export default class VChart extends Vue {
 
             scales: {
                 yAxes: [{
-                    display: false,
+                    display: true,
                     ticks: {
-                        beginAtZero: true
+                        beginAtZero: true,
+                    },
+                    gridLines: {
+                        borderDash: [2, 5],
+                        drawBorder: false
                     }
                 }],
                 xAxes: [{
                     display: true,
                     ticks: {
                         fontFamily: 'font_regular',
-                        autoSkip: true,
+                        autoSkip: false,
                         maxRotation: 0,
                         minRotation: 0,
+                        callback: function(value, index, values): string | undefined {
+                            const valuesLength = values.length;
+                            const firstDateValue = values[0];
+                            const lastDateValue = values[valuesLength - 1];
+                            const isDateValueFirstOrLast = value === firstDateValue || value === lastDateValue;
+                            const middleDateValue = valuesLength / 2;
+                            const isDateValueInMiddleInEvenAmount = index === (middleDateValue - 1);
+                            const isDateValueInMiddleInNotEvenAmount = index === (Math.floor(middleDateValue));
+                            const isAfterEighthDayOfTheMonth = valuesLength > 8 && valuesLength <= 31;
```

Comment:
This numbers should be moved to const with meaning description.
Avoid using magic numbers